RELEASENOTES.docu

```diff
@@ -1689,4 +1689,7 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
       been changed. The old serialized representation is still accepted for
       deserialization in order to preserve checkpoint compatibility.
   </add-note></build-id>
+  <build-id value="next"><add-note> Added support for non-constant
+      compound initializers in contexts where both non-constant initializers
+      and compound initializers are allowed.</add-note></build-id>
```

Comment:
Hard to read, in particular for someone who doesn't know exactly the previous limitations. Replace second half of sentence with a simple example, like `local buffer_t data = {size, new uint8[size]};`?

Reply:
I don't agree that this is hard to read, and I disagree with scrapping the latter half. However, I'm ok with adding an example.

Reply:
Few in the audience know by heart in which contexts non-constant initializers and compound initializers are allowed. What I'm missing (apart from example) is a description of the limitation that we lift. Like, after the example, say "previously, only constant values were permitted in compound initializers". This would probably make some of the original sentence redundant.

Reply:
Ok, alright. Latter half scrapped, example added.